asn1/LedgerLedgerLotocol.asn

```diff
@@ -107,18 +107,27 @@ CALL ::= CLASS {
 CallSet CALL ::= {
     -- All requests MUST use odd typeIds
     -- All responses MUST use even typeIds
-    {1 PacketRequest} |
-    {2 PacketResponse} |
-    {3 PrepareRequest} |
-    {4 PrepareResponse} |
-    {5 FulfillRequest} |
-    {6 FulfillResponse} |
-    {7 RejectRequest} |
-    {8 RejectResponse} |
-    {9 TransferStateRequest} |
-    {10 TransferStateResponse} |
-    {11 CustomRequest} |
-    {12 CustomResponse}
+    -- {1 reserved because this is the typeId of InterledgerProtocolPaymentPacket in
+    -- https://github.com/interledger/rfcs/edit/mj-lll-packet-types/asn1/GenericPacket.asn } |
+    {2 QuoteLiquidityRequest} |
+    {3 QuoteLiquidityResponse} |
+    {4 QuoteBySourceAmountRequest} |
+    {5 QuoteBySourceAmountResponse} |
+    {6 QuoteByDestinationAmountRequest} |
+    {7 QuoteByDestinationAmountResponse} |
+    {8 InterledgerProtocolError} |
+    -- skipping numbering to allow more future ILP-layer packet types
+    {254 CustomRequest} |
+    {255 CustomResponse} |
+    -- rest of types are outside the ILP-layer, so including the ledger layer
+    {256 PrepareRequest} |
```

Comment:
If we reserve type 255 as an escape code, then it's actually safe to number these ledger-level packet types as 9-16, and then everything will just be simple for the near future, until maybe one day we *need* to use two-byte types. And even if that future comes, we will know we'll be prepared for it.